assets/main.css

```diff
@@ -24,7 +24,7 @@ body {
     text-shadow: none;
     border: 1px dashed black;
     padding: 5px;
-    padding-bottom: 5%;
+    padding-bottom: 50px;
```

Comment:
I think the previous percent solution was better since the element size is relative.

Reply:
Yeah even I thought same